### PR TITLE
alt-codes.net

### PR DESCRIPTION
**Pull request by piquark6046**
### To avoid invalid pull requests, please check and confirm following terms

- [X] This is not an ad/bug report;
- [X] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [X] I have performed a self-review of my own changes;
- [X] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [X] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

 - https://github.com/AdguardTeam/AdguardFilters/issues/138617

### Add your comment and screenshots

 - https://www.similarweb.com/website/alt-codes.net/#traffic

### Terms

- [X] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: BaseFilter/sections/specific.txt [hunk 1]
```diff
@@ -277,6 +277,7 @@ chortle.co.uk,coloring.ws,dltk-holidays.com,dltk-kids.com,kidzone.ws,pcsteps.com
 !
 !
 !
+alt-codes.net###copyModal .modal-body
```
Comment (zloyden) [inline BaseFilter/sections/specific.txt:280]: Looks ok, but there is no tag in `.modal-body`. I see only `div.modal-body`.

**Comment by piquark6046** [comment]
It will be fixed by https://github.com/easylist/easylist/pull/14525.

**Comment by Yuki2718** [comment]
I suggest not to close PR until my PR is actually merged into EL. Sometimes it takes ages.

File: BaseFilter/sections/specific.txt (hunk 1 above)
Comment (piquark6046) [reply BaseFilter/sections/specific.txt:280]: Yes. But, I added it for additional safety.

**Review by zloyden — APPROVED** [review]

**Pull request merged**